packages/app/src/messages/actions/resend-message.action.test.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import createMockStore from "redux-mock-store"
+import thunk from "redux-thunk"
+import { AnyAction } from "@reduxjs/toolkit"
+import { resendMessage } from "App/messages/actions/resend-message.action"
+import {
+  RequestResponse,
+  RequestResponseStatus,
+} from "App/core/types/request-response.interface"
+import { resendMessageRequest } from "App/messages/requests/resend-message.request"
+import { ResendMessageError } from "App/messages/errors"
+import { testError } from "App/__deprecated__/renderer/store/constants"
+// import { Message } from "App/messages/dto"
+import { MessageType } from "App/messages/constants"
+import { CreateMessageDataResponse } from "App/messages/services"
+
+jest.mock("App/messages/requests/resend-message.request")
+
+const mockReSendedMessageData: CreateMessageDataResponse = {
+  messageParts: [
+    {
+      message: {
+        id: "6",
+        date: new Date(),
+        content:
+          "Nulla itaque laborum delectus a id aliquam quod. Voluptas molestiae sit excepturi voluptas fuga cupiditate.",
+        threadId: "1",
+        phoneNumber: "+48500600700",
+        messageType: MessageType.OUTBOX,
+      },
+    },
+  ],
+}
+
+const successDeviceResponse: RequestResponse<CreateMessageDataResponse> = {
+  status: RequestResponseStatus.Ok,
+  data: mockReSendedMessageData,
+}
+
+const successDeviceResponseWithEmptyData: RequestResponse<CreateMessageDataResponse> =
+  {
+    status: RequestResponseStatus.Ok,
+  }
+
+const errorDeviceResponseWithMessage: RequestResponse = {
+  status: RequestResponseStatus.Error,
+  error: {
+    message: "I'm error",
+  },
+}
+
+const errorDeviceResponseWithoutMessage: RequestResponse = {
+  status: RequestResponseStatus.Error,
+}
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+describe("when `resendMessage` request return success", () => {
+  test("`resendMessage` returns thread id", async () => {
+    ;(resendMessageRequest as jest.Mock).mockReturnValue(successDeviceResponse)
+    const mockStore = createMockStore([thunk])()
+    const {
+      meta: { requestId },
+    } = await mockStore.dispatch(resendMessage("6") as unknown as AnyAction)
+
+    expect(mockStore.getActions()).toEqual([
+      resendMessage.pending(requestId, "6"),
+      resendMessage.fulfilled(mockReSendedMessageData, requestId, "6"),
+    ])
+
+    expect(resendMessageRequest).toHaveBeenCalled()
+  })
+})
+
+describe("when `resend` request return error", () => {
+  test("fire `resendMessage` returns `rejected` action if error have `message`", async () => {
```

Comment:
Just curious -  why the word `fire` is so often added to the `action` tests? Does it mean something particular in that case? At least for me, it sounds a little unnatural - maybe simply `"resendMessage" returns "rejected" action if error have "message"` or something like in https://github.com/mudita/mudita-center/pull/798#discussion_r903438433?

(this comment refers to other tests in this file as well)